src/Platform/Microsoft.Testing.Platform.MSBuild/Tasks/InvokeTestingPlatformTask.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.Testing.Platform.IPC.Serializers;
 using Microsoft.Testing.Platform.Logging;
 using Microsoft.Testing.Platform.MSBuild.Tasks;
+using Microsoft.Testing.Platform.OutputDevice;
```

Comment:
target framework parser would be nice to move to some other place. or publish, I saw the same need for simplifying the tfm e.g. in the gitub logger.

Won't do here though.